flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adapter/DefaultSchedulingExecutionVertex.java

```diff
@@ -35,21 +32,22 @@
 /**
  * Default implementation of {@link SchedulingExecutionVertex}.
  */
-class DefaultSchedulingExecutionVertex implements SchedulingExecutionVertex {
+class DefaultSchedulingExecutionVertex
+	implements SchedulingExecutionVertex<DefaultSchedulingExecutionVertex, DefaultSchedulingResultPartition> {
 
 	private final ExecutionVertexID executionVertexId;
 
-	private final List<SchedulingResultPartition> consumedPartitions;
+	private final List<DefaultSchedulingResultPartition> consumedPartitions;
```

Comment:
```suggestion
	private final List<DefaultSchedulingResultPartition> consumedResults;
```

Reply:
Done.

I thought `partition` could be also acceptable here because we know they are `DefaultSchedulingResultPartition`s in this implementation. 
But keeping the name in an aligned form might be better.